VirtoCommerce.Platform.Web/Controllers/Api/AuthorizationController.cs

```diff
@@ -74,6 +81,27 @@ public async Task<IActionResult> Exchange(OpenIdConnectRequest request)
 
                 // Create a new authentication ticket.
                 var ticket = await CreateTicketAsync(request, user);
+                var claims = await _userClaimsPrincipalFactory.CreateAsync(user);
+                var limitedPermissions = _authorizationOptions.LimitedCookiePermissions?.Split(';', StringSplitOptions.RemoveEmptyEntries) ?? new string[0];
```

Comment:
The limited permissions delimiter character (`';'`) is used in many places. It should probably be stored to some constant, e.g. somewhere in `PlatformConstants.Security` class.